spec/models/factories_spec.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.factories.map(&:name).each do |factory_name|
```

Comment:
Yeah I think you want `FactoryGirl.lint` which does this for you: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#linting-factories